Library/Homebrew/utils/diskusage.rb

```diff
@@ -0,0 +1,57 @@
+require "find"
+
+module Utils
+  class DiskUsage
```

Comment:
Yes, we could have a common method in `utils.rb` but we'll end up calling that method twice - `pathname.abv` for displaying, and in `cleanup.rb` to get a `disk space usage` value. Or just once if we have a display-cum-update method in `cleanup.rb` without invoking `pathname.abv`.

Here is another [attempt](https://github.com/rakkesh/homebrew/commit/81db726f7dd592667e60ca24099afe5f8a8386ab) to integrate better into `Homebrew` core. I have made changes to `Pathname` to hold `directory size` value, which gets updated when `abv` is invoked but I wonder if this can be done differently.